tools/merger.py

```diff
@@ -51,92 +54,98 @@ def load_merger(filenames,return_mass=False,trim=True):
             if return_mass:
                 out_HM = HM
         else:
-            out_HI = np.hstack((out_HI,HI))
-            out_MP = np.hstack((out_MP,MP))
-            out_MPP = np.hstack((out_MPP,MPP))
-            out_iS = np.hstack((out_iS,iS))
+            out_HI = np.hstack((out_HI, HI))
+            out_MP = np.hstack((out_MP, MP))
+            out_MPP = np.hstack((out_MPP, MPP))
+            out_iS = np.hstack((out_iS, iS))
             if return_mass:
-                out_HM = np.hstack((out_HM,HM))
+                out_HM = np.hstack((out_HM, HM))
 
     if not return_mass:
-        out_HM = np.zeros(len(out_HI))-1
+        out_HM = np.zeros(len(out_HI)) - 1
     return out_HI, out_MP, out_MPP, out_iS, out_HM
 
-def simple_load(filenames,fields):
-    
+
+def simple_load(filenames, fields):
+
     for i in range(len(filenames)):
         fn = filenames[i]
-        print("File number %i of %i"%(i,len(filenames)-1))
-        f = asdf.open(fn,lazy_load=True, copy_arrays=True)
+        print("File number %i of %i" % (i, len(filenames) - 1))
+        f = asdf.open(fn, lazy_load=True, copy_arrays=True)
+        fdata = f['data']
```

Comment:
`f['data']` (or `f.tree['data']`) can be aliased as `fdata` to save a few characters, if desired